tests/DependencyInjection/Compiler/AddDependencyCallsCompilerPassTest.php

```diff
@@ -476,6 +472,9 @@ public function testProcessMultipleOnTopOptionsInServiceDefinition2(): void
         }
     }
 
+    /**
+     * @group legacy
```

Comment:
Why does this one needs the group legacy?

Reply:
We're still passing argument to the constructor.

This will be completely useless in next major version ; I added a comment.